ion_schema_2_0/schema/schema_footer.isl

```diff
@@ -29,43 +29,50 @@ $test::{
 }
 
 $test::{
-  description: "Schema footer must not be present if header is not present.",
-  invalid_schemas: [
+  description: "Everything after the schema footer has no bearing on the schema.",
+  valid_schemas: [
     (
       $ion_schema_2_0
+      schema_header::{}
       type::{
         name: foo
       }
       schema_footer::{}
-    ),
-  ]
-}
 
-$test::{
-  description: "Schema footer must come after schema header.",
-  invalid_schemas: [
+      schema_footer::{}
+    ),
     (
       $ion_schema_2_0
+      schema_header::{}
+      type::{
+        name: foo
+      }
       schema_footer::{}
+
```

Comment:
Looks like this duplicates the previous test

Reply:
It's almost a duplicate, but the previous test case ends with:
```ion
schema_footer::{}
schema_footer::{}
```
and this one ends with:
```ion
schema_footer::{}
schema_header::{}
```